paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/BacsMandateConfirmationFormScreenshotTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.bacs
+
+import com.stripe.android.utils.screenshots.PaparazziRule
+import org.junit.Rule
+import org.junit.Test
+
+class BacsMandateConfirmationFormScreenshotTest {
+    @get:Rule
+    val paparazziRule = PaparazziRule()
```

Comment:
Can we add tests with dark mode, different fonts, and different text scales here?